packages/app-bridge/src/react/useDocumentGroups.ts

```diff
@@ -51,20 +44,31 @@ export const useDocumentGroups = (appBridge: AppBridgeTheme) => {
             });
         };
 
-        window.emitter.on('AppBridge:GuidelineDocumentGroupAction', handleEventUpdates);
```

Comment:
Event still used in clarify

Reply:
That is correct and it should be used. It is used in useDocumentGroups as well.